mne/simulation/tests/test_raw.py

```diff
@@ -428,6 +430,55 @@ def test_simulate_round_trip(raw_data):
         simulate_raw(raw.info, stc, None, None, None, forward=fwd)
 
 
+@pytest.mark.slowtest
+@testing.requires_testing_data
+def test_sourcesim_events(raw_data):
+    """Test of SourceSimulator usage with testing data."""
```

Comment:
can you try to use a more informative description. I am sure we have other tests. If this one is specific to a feature (first_samp?) than let's be explicit. If it's a minor change on an other test, I would just update an other test. Typically I would have started by adding a first_samp in an other test and checked that it was actually tacken into account. I am worried about the time of the mne test suite which is growing too fast

Reply:
I think this warrants its own test.  While I was motivated by the first_samp addition, there was apparently no existing check for the timing of SourceSim events as they appear in the resulting simulated raw object.  That said, I will look more closely at consolidating it into an existing test function (like the one for chpi coils).

Reply:
I simplified the SourceSim test a bit then consolidated it into the raw_bem test function in the same module, as it uses the same testing data.  It takes a fraction of the time that the previous test components take, so that's a plus.  Thanks.  I'll move this from draft to a proper PR.